client/components/domains/domain-suggestion/style.scss

```diff
@@ -104,7 +104,7 @@
 	}
 }
 
-.domain-suggestion__action {
+.button.domain-suggestion__action {
```

Comment:
Did we need the extra css specificity here to override defaults? (Our class naming lets us keep to 1 to 2 classnames for a selector, unless there's an exception).

Reply:
Ah yes, looking at this, we need it for the color override.